Fix accesses modificator

seleniumfan opened the pull request.

seleniumfan committed:
Fix accesses modificator

slawekludwiczak commented on src/main/java/pl/javastart/loan/Config.java:
```diff
@@ -0,0 +1,14 @@
+package pl.javastart.loan;
+
+public class Config {
```
Konfiguracja nie musi być publiczna, nie jest używana poza pakietem

seleniumfan replied:
Wielkie dzięki za code review - super wyjaśnione! 👍

slawekludwiczak reviewed (commented):
code review https://javastartbootcamp.slack.com/archives/C04FUHPRS74/p1675239841xxxx29

seleniumfan committed:
Fix:
- delete public from class Config
- change public to private in config variable
